modules/recurlyjs/recurlyjs.services.yml

```diff
@@ -0,0 +1,6 @@
+services:
+  recurlyjs.route_subscriber:
```

Comment:
I believe this can be anything with the first part being the module name of course.